bubblewrap.c

```diff
@@ -2641,6 +2673,7 @@ main (int    argc,
   int res UNUSED;
   cleanup_free char *args_data UNUSED = NULL;
   int intermediate_pids_sockets[2] = {-1, -1};
+  sigset_t sigmask;
```

Comment:
A better name for this would be something like `sigmask_before_forwarding`, and it should probably be initialized with e.g. `sigemptyset()` rather than leaving it uninitialized.